src/Facility.CodeGen.JavaScript/JavaScriptGenerator.cs

```diff
@@ -109,6 +109,23 @@ public override CodeGenOutput GenerateOutput(ServiceInfo service)
 						typeNames.Add($"I{dtoInfo.Name}");
 						WriteDto(code, dtoInfo, service);
 					}
+
+					foreach (var enumInfo in service.Enums)
+					{
+						typeNames.Add(enumInfo.Name);
+						code.WriteLine();
+						WriteJsDoc(code, enumInfo);
+						using (code.Block($"export enum {enumInfo.Name} {{", "}"))
+						{
+							foreach (var value in enumInfo.Values)
+							{
+								code.WriteLineSkipOnce();
+								WriteJsDoc(code, value);
+								code.WriteLine($@"{value.Name} = ""{value.Name}"",");
```

Comment:
Nit: we use `'` in the other generated code.

Reply:
Updated: https://github.com/FacilityApi/FacilityJavaScript/pull/26/commits/28f54864e1d092063a70eccf0e6a5e35da85ca14